output/clinical_collections.json

```diff
@@ -5,7 +5,7 @@
     "Wiki": true,
     "Format": ".csv",
     "Filenames": "<a href=\"https://wiki.cancerimagingarchive.net/download/attachments/70225642/https://wiki.cancerimagingarchive.net/download/attachments/70225642/UPENN-GBM_clinical_info_v1.1.csv?api=v2https://wiki.cancerimagingarchive.net/download/attachments/70225642/UPENN-GBM_clinical_info_v1.0.csv?api=v2\">UPENN-GBM_clinical_info_v1.1.csv</a>: 2df31ca5353bcccb5bc4baf2159eaafa",
-    "Notes": "new collection detected as of 2022-07-05"
+    "Notes": "new collection detected as of 2022-07-06"
```

Comment:
@G-White-ISB I do not think this note is accurate - the specific collection in this section of file is not new as of this PR. I think we should revisit the content of "Notes" here. What do you think?